.github/workflows/publish-collection.yml

```
@@ -1,6 +1,6 @@
 ---
 name: Publish Ansible Collection
-'on':
+"on":
```

```suggestion
'on':
```
Revert auto-format change